Foundation/Bridging.swift

```diff
@@ -108,10 +89,8 @@ internal final class _SwiftValue : NSObject, NSCopying {
     static func store(_ value: Any) -> NSObject {
         if let val = value as? NSObject {
             return val
-        } else if let val = value as? _ObjectBridgeable {
-            return val._bridgeToAnyObject() as! NSObject
         } else {
-            return _SwiftValue(value)
+            return (value as AnyObject) as! NSObject
```

Comment:
Looks like this change leads to crash in cases like this:
```
class A {}
let array: NSArray = [A()]
```

Reply:
@millenomi Can you look into this https://github.com/apple/swift-corelibs-foundation/pull/2500/files ?